treon/task.py

```diff
@@ -27,14 +27,14 @@ def run_tests(self):
                 result += console_output
 
             LOG.info(result)
-        except Exception:
+        except Exception:  # pylint: disable=broad-except
```

Comment:
Should we disable this for entire repo?

Reply:
No, better we fix it. There is a number of problems with this branch:
* The underlying method raises `CellExecutionError`, which is the proper exception type.
* Said exception already ships a stringified traceback of the cell, which we can simply use. The traceback of treon itself (as formatted by `traceback.format_exc`) is not relevant at this point.
*  The template from `error_string` method adds extra whitespace in the beginning of the traceback, despite `textwrap.dedent`:

```
ERROR in testing nb/Testi.ipynb
            Traceback (most recent call last):
  File "/redacted/treon/treon/task.py", line 23, in run_tests
etc.
```

I'll include a separate commit addressing these concerns.

Reply:
Sounds good. Feel free to make a separate PR for that one.